### ISSUE TITLE
AWS Preparation for submariner addon (duplicate--not worked)

### ISSUE THREAD

**Issue by trujillm** [open-cluster-management/rhacm-docs#2774]
## Product release(s)
- [ACM 2.3 GA] 

## Locations of the suggested changes?

Links:
* https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.3/html/services/services-overview

Section:
* [Preparing Amazon Web Services to deploy Submariner](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.3/html/services/services-overview#preparing-aws) (1.1.4.1.1)

## What are the suggested changes?
Modify the example or explicitly call out that the values needed for aws_access_key_id: <aws-access-key-id> and aws_secret_access_key: <aws-secret-access-key> need to be base64 encoded

Currently in the documentation section Preparing Amazon Web Services to deploy Submariner it outlines the following example for the AWS secret key and secret key ID
```
cat << EOF | oc apply -f -
apiVersion: v1
kind: Secret
metadata:
    name: <managed-cluster-name>-aws-creds
    namespace: <managed-cluster-namespace>
type: Opaque
data:
    aws_access_key_id: <aws-access-key-id>
    aws_secret_access_key: <aws-secret-access-key>
EOF
```
This will fail when the setup begins as the values needed for aws_access_key_id: <aws-access-key-id> and aws_secret_access_key: <aws-secret-access-key> need to be base64 encoded. 

This is not stated anywhere in the documentation and no errors are thrown to indicate the configuration is incorrect.

**Comment by swopebe** [comment]
This should/may be opened on the backlog.

**Comment by cadawson** [comment]
This was resolved in PR #2763 . Closing this issue.
